dom.bs

```diff
@@ -1195,15 +1207,22 @@ for discussion).
   <p>While <var>parent</var> is non-null:</p>
 
   <ol>
+   <li><p>Let <var>relatedTarget</var> be the result of invoking <a>retargeting</a>
+   <var>event</var>'s <a>relatedTarget</a> against <var>parent</var> if <var>event</var>'s
+   <a>relatedTarget</a> is non-null, and null otherwise.
+
    <li><p>If <var>target</var>'s <a for=tree>root</a> is a
    <a>shadow-including inclusive ancestor</a> of <var>parent</var>, then append
-   (<var>parent</var>, null) to <var>event</var>'s <a for=Event>path</a>.
+   (<var>parent</var>, null, <var>relatedTarget</var>) to <var>event</var>'s <a for=Event>path</a>.
+
+   <li><p>Otherwise, if <var>parent</var> and <var>relatedTarget</var> are identical, set
+   <var>parent</var> to null.
```

Comment:
"then set _parent_ to null"

The convention is to use "then" after the comma following an if statement.